pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -425,10 +425,12 @@
   :-  ~
   %-  as-octs:mimes:html
   %-  crip
-  %-  zing
+  %-  zing  ^-  ^wall
+  %-  zing  ^-  (list ^wall)
   %+  turn  wall
   |=  t=tape
-  "{t}\0a"
+  ^-  ^wall
+  ~[t "\0a"]
```

Comment:
It seems like a `+weld` here might be more efficient that a `+zing` later. But it probably doesn't matter compared to the impact of removing the tape interpolation.

Reply:
Can you explain the heuristic you're using from a performance perspective of when it's better to use `+zing` vs `+weld` ?

Reply:
Isn't most of the impact of the tape interpolation the weld? The zing seems better because it'll traverse the `t` list only once.